### DISCUSSION TITLE
為何現在的華語音樂相較於十年前被認為退步了

### DISCUSSION THREAD

**Discussion by uwje** [1101-GS4719-C/Python-Community-GS4719-C#1732] [General]
隨著時間發展，事情或能力應該要跟著進步，但最近的華語音樂卻被認為是退步了，我認為有以下幾點造成這個現況：
1.網路的進步，寫歌與傳播歌曲的渠道變得比較容易，使得許多人皆可創作，雖然有更多歌曲能被更多人聽到，但也會出現品質參差不齊的狀況出現。
2.大量抄襲與重新創作的出現，許多人為了要紅，會把許多歌曲串接在一起，聲稱是自己的歌曲。
3.獲得資訊的渠道增多，讓我感覺現在的歌詞寫得較為平淡，無內容含量。
4.重複或相似的旋律，無內容的歌詞。
我認為在現在的時代，人變的過於浮躁，以前的歌手拼盡全力才獲得的機會，現在卻可以以較為輕易的方式獲取，使得現在的人不用去精進自己來獲得觀眾的喜愛，反正去找尋更快或流行的方式來獲取觀眾的眼光。

**Comment by sorn32** [comment]
我覺得很神奇的是，以前那個年代的歌，現在聽都還是覺得很好聽ㄟ(*´∀`)~♥
現在的抖音歌感覺結構都差不多╮(╯ _ ╰)╭

**Comment by 111426027** [comment]
喜歡十年前的華語歌+1，覺得現在的流行歌都不耐聽，聽一陣子就膩了，但是以前的歌現在聽依舊覺得是神曲!

**Comment by Terryyyyyyyyyyyy** [comment]
雖然現在的歌手也會創作出好聽且有深度的歌，但受到其他奇形怪狀歌曲的影響，導致他們的歌曲不受重視，希望那些抄襲的人，可以精進自己，創作出屬於自己的歌

**Comment by THH1118** [comment]
可能是現在的歌手不是只專注在做歌，也會去當主持人、拍綜藝出外景甚至兼著做Youtuber，所以品質下降了吧，以前最多只會看到歌手去一些訪談節目宣傳歌曲而已

**Comment by WeiYen1103** [comment]
我覺得會有這樣的想法是因為，近期的歌人聲都偏小聲，背景的電子音反而都蓋過人聲，加上最近的歌都蠻芭樂的QQ，我覺得相較之前少了很多特色，都是用幾個和弦重複構成，我現在也還是比較常聽幾年前的歌XD

**Comment by Ryanlolry** [comment]
我認為真正的歌手仍然在認真的做歌，但市場上充斥著太多不認真的、把做音樂當影片剪接，所以整個行業被搞得烏煙瘴氣，導致認真做事的歌手被掩蓋。

**Comment by Patsy17** [comment]
我覺得現在的歌都為了更迎合聽眾的喜好，越來越芭樂歌QQ
而且好像只要有錢就可以出歌了～

**Comment by davidteng00000** [comment]
華語音樂發展複雜，多因素影響。數位平台優點是提供更多創作機會，但也引發品質不一問題。抄襲、歌詞內容減少、重複旋律等是常見議題。技術進步降低創作門檻，但也提高競爭。或許多元的創作風格可帶來轉變。
